tests/i18n/tests.py

```diff
@@ -1661,6 +1689,16 @@ def test_get_language_from_request(self):
         lang = get_language_from_request(r)
         self.assertEqual('bg', lang)
 
+    def test_get_language_from_request_null(self):
+        r = self.rf.get('/')
```

Comment:
It would be fine to call the function with `None` since `r` isn't used. Again, use `self.settings` to verify that `settings.LANGUAGE_CODE` is returned.

Reply:
Same as above I used `@override_settings`